### ISSUE TITLE
Problem with conversion to has

### ISSUE THREAD

**Issue by TriderG75** [yaourdt/mgos-to-tasmota#73]
Hi, using this command http://shellyip/ota?url=http://dl.dasker.eu/firmware/mg2haa-Shelly1.zip, instead appearing the hotspot haa-xxxxxx it opens a mongoose page and the shelly 1 stops to work. Using the mg2tasmota command, all is ok. I add the photo of the mongoose page I'm talking about. 
Thanks

![image](https://user-images.githubusercontent.com/62612806/161447153-36c7c101-87ac-4683-a346-9adc86012027.png)

**Comment by getinmybelly** [comment]
Same problem here, though with the Dimmer 2 update. 

![3DCB8E5A-C04A-4A52-B101-112547237258](https://user-images.githubusercontent.com/71928532/163310131-843b80d6-58ae-4ec8-a92e-376c94ed03f2.jpeg)

**Comment by jamr200** [comment]
I have the same problem with shelly1 and shelly 2.5

**Comment by Tibi-rvb** [comment]
Same for me with a RGBW2

**Comment by marcoscv-work** [comment]
Same thing here!
do you have any news about this?

**Comment by RavenSystem** [comment]
@yaourdt GitHub did some changes in their SSL certificates and your firmware are not able to download files from its servers. Probably, a new build, compiled with latest SDK, is needed.

**Comment by michapi** [comment]
Is there a way to recover the shelly?

**Comment by mallito1307** [comment]
Same issue here!
Please solve it 🙏🏻

**Comment by tokamac** [comment]
Wow, your program "installing" (failing to) HAA actually bricked the two Shelly Dimmer 2 modules I bought. Now they are impossible to reach or update. And you won't even provide an update, yet alone make a warning statement about HAA on the main install page. Shame. Refund! Ahah juste joking but you seriously pissed me off.

**Comment by TriderG75** [comment]
> statement

You can use the method two messages above to recover the Shellys. Of course you have to use the correct file for your Shelly

**Comment by tokamac** [comment]
Thanks, that worked! Using SHDM-2.zip for the Dimmer 2. I see that RavenSystem advises the following in the Third-Party Installation Tools on the HAA install page, for Shelly:

> [Shelly Devices OTA Version](https://github.com/yaourdt/mgos-to-tasmota)
IMPORTANT: Until developer fixes [current issue](https://github.com/yaourdt/mgos-to-tasmota/issues/73#issuecomment-1124240911), install Tasmota, then [Tasmota Minimal](http://ota.tasmota.com/tasmota/tasmota-minimal.bin.gz) from WebUI, and then [fullhaaboot.bin](https://github.com/RavenSystem/haa/releases/latest/download/fullhaaboot.bin) from WebUI.

I put the solution here since it's been weeks this annoying bug exists but the developper won't step in to make at least a statement.

**Comment by abishur** [comment]
dumb question here, but when you ran the http://shellyip/ota?url=http://dl.dasker.eu/firmware/mg2haa-Shelly1.zip command did you replace the "shellyip" with the ip of your shelly?

**Comment by TriderG75** [comment]
> dumb question here, but when you ran the http://shellyip/ota?url=http://dl.dasker.eu/firmware/mg2haa-Shelly1.zip command did you replace the "shellyip" with the ip of your shelly?

Of course. If you don't put the ip the command doesn't work and the Shelly remains with the original fw. In this case something changes in the fw and "bricks" the Shelly. Fortunately there is a way to recover it (look some messages above).

**Comment by arkn666l** [comment]
for me works today, yes i saw this page .. index of.. but minutes later the tasmota wifi appears...
